openweather.js

```diff
@@ -131,6 +133,16 @@ function onPositionUpdate(value, callback) {
 
         owm.setCoordinate(value.latitude, value.longitude);
         log("OWM Coordinates "+value.latitude+","+value.longitude);
+
+        // get the Temperature  
+	    owm.getTemperature(function(err, temp){
```

Comment:
This will return exactly the same data as getSmartJSON further down below

Reply:
I didn't realize. will fix.

Reply:
fixed.